Loenn/entities/anxietyorigin.lua

```diff
@@ -1,13 +1,20 @@
-local default = {}
+local origin = {}
 
-default.name = "ContortHelper/AnxietyOriginController"
-default.depth = 0
-default.texture = "ahorn/muntheory/contorthelper/anxietyorigincontroller"
-default.placements = {
+origin.name = "ContortHelper/AnxietyOriginController"
+origin.depth = 0
+origin.texture = "ahorn/muntheory/contorthelper/anxietyorigincontroller"
+origin.placements = {
     name = "default",
     data = {
         origin = "Player"
     }
 }
 
-return default
\ No newline at end of file
+origin.fieldInformation = {
+    origin = {
+        options = { "Player" },

```

Comment:
There is a table defined the in Contort Helper library for this. I know it's only one value, but it'd help in the future should new values be added to that table. 
```suggestion
        options = contorthelper.origintypes, -- make sure to require contorthelper
```